data/publiccloud/terraform/azure_nfstest.tf

```diff
@@ -168,10 +168,6 @@ resource "azurerm_storage_account_network_rules" "openqa-group" {
   virtual_network_subnet_ids = [azurerm_subnet.openqa-subnet.id]
   // AZURE LIMITATION: After setting Deny, we need to allow this host otherwise we cannot do changes or delete the resources
   ip_rules = [chomp(data.http.myip.response_body)]
-  
-  private_link_access {
-    endpoint_resource_id     = azurerm_subnet.openqa-subnet.id
```

Comment:
We need to check if this is really not required. I remember that when using NFS there are some network one restrictions. Let's make a VR that shows it works.

Reply:
Marking it as "requesting changes" because this has the potential to break test runs (in a different way).

A valid VR is enough to resolve this

Reply:
+1 to what Felix said . it looks weird to me that we **removing** something I would expect that some property changed format due to not backward-compatible changes but not something like that . Anyway if you will provide passed VR AND some brief explanations why this is suppose to be a fix I will add +1